MetadataExtractor/IO/BufferReader.Sequential.cs

```diff
@@ -114,17 +114,19 @@ public ulong GetUInt64()
 
     public string GetString(int bytesRequested, Encoding encoding)
     {
-        // This check is important on .NET Framework
```

Comment:
Could we keep this comment on the zero check please? It still exists in other methods.